ssv/spectest/tests/runner/consensus/no_running_consensus_instance.go

```diff
@@ -19,17 +20,20 @@ func NoRunningConsensusInstance() tests.SpecTest {
 				Duty:   &testingutils.TestingSyncCommitteeContributionDuty,
 				Messages: []*types.SSVMessage{
 					testingutils.SSVMsgSyncCommitteeContribution(
-						testingutils.TestingProposalMessageWithIdentifierAndFullData(
-							ks.Shares[4], types.OperatorID(4), testingutils.SyncCommitteeContributionMsgID,
-							testingutils.TestSyncCommitteeContributionConsensusDataByts,
+						testingutils.TestingPrepareMessageWithParams(
+							ks.Shares[1],
+							1,
+							qbft.FirstRound,
+							qbft.FirstHeight,
+							testingutils.SyncCommitteeContributionMsgID,
+							testingutils.TestingQBFTRootData,
 						),
 						nil),
 				},
-				PostDutyRunnerStateRoot: "29862cc6054edc8547efcb5ae753290971d664b9c39768503b4d66e1b52ecb06",
+				PostDutyRunnerStateRoot: "b2e883cc81caaed04f3e40e8561ae55aa1f6abcdb3168e5cc5c834b1d327026e",
 				OutputMessages: []*types.SignedPartialSignatureMessage{
 					testingutils.PreConsensusContributionProofMsg(ks.Shares[1], ks.Shares[1], 1, 1),
```

Comment:
I've removed this test as it's redundant.

In the past it tested cases in which a consensus message is received but no running qbft instance is present. 
After a few changes in the code (over the past few months) in such cases it will either be a future message or carrying a pre-consensus in which case a qbft instance will be started. 

Essentially it doesn't test anything anymore